components/directory-size-exporter/utils/fs.go

```diff
@@ -0,0 +1,52 @@
+package utils
```

Comment:
Since there is only one `exporter_test.go` left over, I suggest to move all the test helpers from this package  (fs and prometheus) also to the test file. I don't think we need `fs_test.go`. Having tests for the test helpers is a bit too much.